src/main/java/com/thoughtworks/gauge/GaugeConstant.java

```diff
@@ -21,6 +21,7 @@
     public static final String GAUGE_PROJECT_ROOT = "GAUGE_PROJECT_ROOT";
     public static final String GAUGE_CUSTOM_COMPILE_DIR = "gauge_custom_compile_dir";
     public static final String DEFAULT_SRC_DIR = "src/test/java";
+    public static final String DEFAULT_SRC_DIRS = String.format("src/main/java,%s", DEFAULT_SRC_DIR);
```

Comment:
Please initialize this as a static array instead of using a string format 
```suggestion
    public static final String[] DEFAULT_SRC_DIRS = { "src/main/java", DEFAULT_SRC_DIR };
```